apps/indexer/test/indexer/block/fetcher_test.exs

```diff
@@ -449,7 +451,9 @@ defmodule Indexer.Block.FetcherTest do
                      "payments" => [
                        %{
                          "address" => fee_payment_address_hash,
-                         "value" => value_string_1
+                         "value" => value_string_1,
+                         "valueFromFees" => value_from_fees_string,
+                         "valueFromMainchain" => value_from_mainchain_string
```

Comment:
Ideally we would test both scenarios where the response has valueFromFees/valueFromMainchain and not

Reply:
I added a second fee payment with no mainchain reward and the related assert statements.
Unfortunately I encountered some problems (related to rpc method mocking) in adding a new address to the test so I use the already existing address of the variable `fee_payment_address_hash` to manage also the second fee payment item.
However the insert or more fee payments related to different addresses is extensively tested in the v2 unit test of the branch [WEB3-659](https://github.com/HorizenLabs/eon-explorer/tree/WEB3-659)